sdccc/src/main/java/com/draeger/medical/sdccc/sdcri/testclient/TestClientImpl.java

```diff
@@ -318,16 +344,29 @@ void deviceFound(final ProbedDeviceFoundMessage message) {
                 throw new IOException(COULDN_T_CONNECT_TO_TARGET);
             }
         } catch (final InterruptedException | TimeoutException | ExecutionException e) {
-            LOG.error(
-                    "Couldn't find a device with {}, {}, {}, {}, {}, {} and {}",
-                    this.eprSearchLogString,
-                    this.facilitySearchLogString,
-                    this.buildingSearchLogString,
-                    this.pointOfCareSearchLogString,
-                    this.floorSearchLogString,
-                    this.roomSearchLogString,
-                    this.bedSearchLogString,
-                    e);
+            if (inReconnectProcess.get()) {
```

Comment:
inReconnectProcess would probably have to be used in many more cases and functions similiarly to how it is used here and also it would have to be thread id aware.

For example I got a TimeoutException in https://github.com/Draegerwerk/SDCcc/pull/216/files#diff-955d82d123e93ce4675e6524a4b941f007756b1589cfb87114b82fffa9e2d181R380 during a reconnect and although the reconnect was successful on the second attempt, the test run still got invalidated.

Reply:
Added a whitelist to the log appender to not invalid the test run during reconnect.